astropy/visualization/wcsaxes/formatter_locator.py

```diff
@@ -262,6 +263,10 @@ def format(self, value):
         if value is None:
             return
 
+        self._alwayssign = value.startswith("+")
+        if self._alwayssign:
```

Comment:
At face value, this can look like a prime candidate for using the walrus operator, but Python does not allow the walrus operator to assign to instance attributes.